tests/kernel/device/kernel.device.pm fails on bt510/bt6x0

`./scripts/twister -s tests/kernel/device/kernel.device.pm -p bt510 -p bt6x0`

gets:

```
[ 77%] Building C object zephyr/drivers/sensor/si7055/CMakeFiles/drivers__sensor__si7055.dir/si7055.c.obj
In file included from /home/galak/git/zephyr/include/kernel_includes.h:21,
                 from /home/galak/git/zephyr/include/kernel.h:17,
                 from /home/galak/git/zephyr/include/init.h:11,
                 from /home/galak/git/zephyr/include/device.h:29,
                 from /home/galak/git/zephyr/include/drivers/sensor.h:23,
                 from /home/galak/git/zephyr/drivers/sensor/si7055/si7055.c:10:
/home/galak/git/zephyr/twister-out/bt510/tests/kernel/device/kernel.device.pm/zephyr/include/generated/devicetree_unfixed.h:7232:35: error: 'DT_N_S_soc_S_i2c_40003000_S_si7055_40_P_wakeup_source' undeclared here (not in a function); did you mean 'DT_N_S_soc_S_i2c_40003000_S_lis2dh12_18_P_wakeup_source'?
      |                                   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

--- a/dts/bindings/sensor/silabs,si7055.yaml
+++ b/dts/bindings/sensor/silabs,si7055.yaml
@@ -1,0 +1,8 @@
+# Copyright (c) 2021, Linaro ltd
+# SPDX-License-Identifier: Apache-2.0
+
+description: Si7055 temperature sensor
+
+compatible: "silabs,si7055"
+
+include: i2c-device.yaml

